edxval/tests/test_views.py

```diff
@@ -813,6 +813,7 @@ def test_create_transcript(self):
         Tests POSTing transcript successfully.
         """
         post_transcript_data = dict(self.transcript_data)
+        post_transcript_data.pop('file_data')
```

Comment:
why this change?

Reply:
`post_transcript_data` has `file_data` which we added to use for content. This isn't needed in this test. This is already happening in case of `video_id` as well.